x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -72,6 +72,7 @@ export const createSecurityRuleTypeWrapper: CreateSecurityRuleTypeWrapper =
             state,
             updatedBy: updatedByUser,
             rule,
+            isPreview,
```

Comment:
We should pass this in as a param to `createSecurityRuleTypeWrapper` rather than the executor directly to avoid adding extra params to the global RuleExecutorOptions type.

Reply:
ya, I figured there was probably another way to send that in, without modifying the alerting signature, and that sounds about right.

Reply:
done